integration-tests/scaffolding/rootHooks.ts

```diff
@@ -1,19 +1,29 @@
 import { ExtrinsicHelper } from "./extrinsicHelpers";
 import { createKeys } from "./apiConnection";
-import { devAccounts } from "./helpers";
+import { devAccounts, rococoAccounts } from "./helpers";
 
 export let EXISTENTIAL_DEPOSIT: bigint;
 
 exports.mochaHooks = {
     async beforeAll() {
         await ExtrinsicHelper.initialize();
-        for (const uri of ["//Alice", "//Bob", "//Charlie", "//Dave", "//Eve", "//Ferdie"]) {
-            devAccounts.push({
-                uri,
-                keys: createKeys(uri),
-            });
+        let default_source;
+        if (process.env.CHAIN_ENVIRONMENT == "rococo") {
+            const seed_phrase = "Place holder for seed phrase";
+            rococoAccounts.push({
```

Comment:
Is there a reason this isbn't just pushing into `devAccounts`?

Reply:
The reason is for clarity; to differentiate between the common accounts available in a development environment vs the accounts available for testnet.